From 8e2c2ca45148968a37f93741035e7e648ad1eab5 Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Thu, 6 Mar 2025 16:22:10 +0100
Subject: [PATCH] Drop SVGPathElement interface

The interface is more thoroughly defined in SVG Paths and now implemented in
Firefox.

No related issue, pending clarification on the status of SVG Paths and SVG 2.
---
 ed/idl/SVG.idl | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/ed/idl/SVG.idl b/ed/idl/SVG.idl
index 9ce619d1e..ca918802e 100644
--- a/ed/idl/SVG.idl
+++ b/ed/idl/SVG.idl
@@ -419,10 +419,6 @@ interface SVGAnimatedPreserveAspectRatio {
   [SameObject] readonly attribute SVGPreserveAspectRatio animVal;
 };
 
-[Exposed=Window]
-interface SVGPathElement : SVGGeometryElement {
-};
-
 [Exposed=Window]
 interface SVGRectElement : SVGGeometryElement {
   [SameObject] readonly attribute SVGAnimatedLength x;
-- 
2.37.1.windows.1

